opts/envfile.go

```diff
@@ -4,9 +4,16 @@ import (
 	"bufio"
 	"fmt"
 	"os"
+	"regexp"
 	"strings"
 )
 
+var (
+	// Variables set by the user must have a name consisting solely of
+	// alphabetics, numerics, and underscores - the first of which must not be numeric.
+	EnvironmentVariableRegexp = regexp.MustCompile("^[[:alpha:]][[:alpha:][:digit:]_]+$")
```

Comment:
@duglin ohh right, I assumed it had to start with a *alpha* char, but reading back the comment I put, yep I missed the case of a variable starting by ``_``. I'll update it.